app/selectors/post_list.js

```diff
@@ -66,7 +66,11 @@ export function makePreparePostIdsForPostList() {
                 }
 
                 // Only add the new messages line if a lastViewedAt time is set
-                const postIsUnread = post.create_at > lastViewedAt && post.user_id !== currentUser.id;
+                let postIsUnread = post.create_at > lastViewedAt && post.user_id !== currentUser.id;
+                if (lastViewedAt === 0 && post.type === Posts.POST_TYPES.COMBINED_USER_ACTIVITY) {
+                    postIsUnread = false;
+                }
+
                 if (lastViewedAt !== null && !addedNewMessagesIndicator && postIsUnread && indicateNewMessages) {
```

Comment:
I think we might just need to change this check to see if lastViewedAt is nonzero instead of checking it twice. We only want to see this line if we've seen the channel before, so the post type doesn't matter here. As it is, we're checking lastViewedAt twice

Reply:
Got it, please see updated version.  Thanks!